src/components/useIncrementalResponse.test.ts

```diff
@@ -0,0 +1,170 @@
+import { act, renderHook } from '@testing-library/react-hooks'
+import _ from 'lodash/fp'
+import useIncrementalResponse from 'src/components/useIncrementalResponse'
+import IncrementalResponse from 'src/libs/ajax/IncrementalResponse'
+import { ErrorState, LoadingState, ReadyState } from 'src/libs/type-utils/LoadedState'
+
```

Comment:
lets move this hook and test file to /src/components/common/hooks directory.  I hope to move our basic components (buttons, dropdown, etc) to /src/components/common/basic... or something like that at some point to cut down on the number of files directly in the /src/components folder.

Reply:
Moved in https://github.com/DataBiosphere/terra-ui/pull/3468/commits/4f49b42c268c783e38e57b5918ba7d0947b62ec8.